.github/workflows/run-scenes-e2e.yml

```diff
@@ -0,0 +1,41 @@
+name: Run dashboard scenes e2e
+
```

Comment:
Previous test runs of this workflow https://github.com/grafana/grafana/actions/runs/8829700520/job/24241279325?pr=84598